freetextresponse/locale/vi/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,41 @@
+# PLACE HOLDER FILE, 
+# only the mo is used but this is where I found a translated string to test.
```

Comment:
What do you mean by "placeholder file"?
`.mo` files are the binary translation file that is used at runtime,
but it's created _from_ the `.po` file.